lib/cli/src/frameworks/angular/button.component.ts

```diff
@@ -37,7 +37,7 @@ export default class ButtonComponent {
    * @required
    */
   @Input()
-  label: string;
+  label = 'Button';
```

Comment:
Why can't this just be a required string?

Reply:
Also, welcome back!! 🎉

Reply:
@shilman you mean optional? I was going to make it optional but the `@required` 2 lines above and it scared me 😱 😄 .

Reply:
Why can't it be required and not have a default value?

Reply:
Because of `strictPropertyInitialization` (enabled in strict mode): https://devblogs.microsoft.com/typescript/announcing-typescript-2-7/#stricter-class-property-checks

Reply:
Thanks for that. TypeScript 🤦

Reply:
Well, this actually makes sense. The Angular compiler does not force you to set required Inputs in your template. Thus, even if you claim that an Input is required, at runtime it might be undefined if you don't assign a default value.

On the other hand, even if the Angular compiler enforced this, how would the framework-agnostic Typescript compiler know this?

So, this problem arises from the way of how components are built upon Typescript classes.